src/components/binding/binding-panel.js

```diff
@@ -160,10 +160,10 @@ genotet.BindingPanel.prototype.initChrs_ = function() {
 
 /**
  * Adds a select2 for a new track.
+ * @param {number} trackIndex Index of the added track.
  * @private
  */
-genotet.BindingPanel.prototype.addTrack_ = function() {
-  var trackIndex = this.data.tracks.length - 1;
+genotet.BindingPanel.prototype.addTrack_ = function(trackIndex) {
```

Comment:
When you write addTrack_(trackIndex) you should only consider this function itself, rather than how other functions will call it. Otherwise you create complex dependencies which makes debugging extremely hard later.

Now we have addTrack_(trackIndex), which is supposed to create a select2 for track at index trackIndex. Therefore, the select2 UI it creates should be placed at the correct index. However right now it is not the case.